examples/asr/asr_adapters/eval_asr_adapter.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+# Evaluate an adapted model
+
+python eval_asr_adapter.py \
```

Comment:
Yes, because during inference we dont want to set any value for the train or validation dataloaders and waste CPU memory. I could automate this in future versions of the script, but for now, lets keep it simple.